src/toolbars/examples/toolbar.js

```diff
@@ -488,6 +520,15 @@
         }
       };
 
+      $scope.noItemsConfig = {
+        title: 'No results match.',
```

Comment:
Pls change to match verbiage in [pf core](http://www.patternfly.org/pattern-library/forms-and-controls/filter/#design) (null result set at bottom of page) "No results match the filter criteria"  -thanks

Reply:
Completed, thanks!